src/Adapter/Category/Repository/CategoryRepository.php

```diff
@@ -63,6 +64,25 @@ public function __construct(
         $this->dbPrefix = $dbPrefix;
     }
 
+    /**
+     * @param CategoryId $categoryId
+     *
+     * @return Category
+     *
+     * @throws CategoryNotFoundException
+     */
+    public function get(CategoryId $categoryId): Category
```

Comment:
Could you add an integration test for checking the exception ?

Reply:
It's not needed as it's part of the `AbstractObjectModelRepository` which is already used everywhere If you want to add unit/integration tests on this abstract class you can create a dedicated PR but it's not the scope of this one

Besides this `get` method is indirectly tested via the behat tests for `GetProductForEditing`